PSIPro.ino

```diff
@@ -156,27 +162,28 @@
  *  Command xPy - Sets various board parameters.
  *                If x is 0, Set the alwaysOn behavior of the panel
  *                  The default mode for the panel is to display command sequences for 
- *                  a given time, then revert to the standard swipe behavior.  
+ *                  a given time, then revert to the default pattern.  
  *                  By sending the xPy command, this can be changed.
  *                  Y is either 0 or 1 (default or always on mode)
- *                  0P0 - Default mode, where swipe is restored after the sequence plays
+ *                  0P0 - Default mode, where default pattern is restored after the sequence plays
```

Comment:
let's say default pattern (swipe) since the standard script uses swipe for the default, and changing it isn't really officially supported, even if it works in some cases.

Reply:
Will do.